opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraph.jsx

```diff
@@ -1497,78 +1567,60 @@ class InvestigationGraphComponent extends Component {
   }
 
   async handleDeleteSelected() {
```

Comment:
Some rework in this function to make one query with all the ids of entities to remove instead of multiple queries.
It seems that making multiple queries created issues as some were not executed (ie. when removing the selected node, both the ids of the node entity and associated relationship were sent for removal but either one or the other was not removed in ES, sometimes the node, sometimes the relationship from a test to an other).